charts/sck-otel/templates/_config.tpl

```diff
@@ -368,4 +376,22 @@ service:
         {{- end }}
         {{- end }}
     {{- end }}
+    {{- if .Values.journaldLogs.enabled }}
+    logs/journald:
+      receivers:
+        - journald
+      processors:
+        - batch
+        - resource/splunk
+      exporters:
+        {{- if .Values.splunkPlatform.endpoint }}
```

Comment:
could you rebase and use `{{- if eq (include "splunk-otel-collector.sendLogsToSplunk" .) "true" }}`